index.html

```diff
@@ -2,6 +2,11 @@
 <html>
 <head>
   <title>Kinto React Boilerplate</title>
+  <style>
+    .disabled {
+      color: GrayText;
+    }
+  </style>
```

Comment:
How about moving this to some external stylesheet? It's a boilerplate after all.